pkg/manifest/raw_bootc.go

```diff
@@ -199,7 +204,18 @@ func (p *RawBootcImage) serialize() osbuild.Pipeline {
 		usersStage.Mounts = mounts
 		usersStage.Devices = devices
 		pipeline.AddStage(usersStage)
+	}
+
+	// First create custom directories, because some of the custom files may depend on them
+	if len(p.Directories) > 0 {
+		pipeline.AddStages(osbuild.GenDirectoryNodesStages(p.Directories)...)
+	}
+
+	if len(p.Files) > 0 {
+		pipeline.AddStages(osbuild.GenFileNodesStages(p.Files)...)
+	}
 
+	if len(p.Users) > 0 || len(p.Files) > 0 || len(p.Directories) > 0 {
```

Comment:
Why did this become conditional?  We're always writing `/etc/fstab`, so that should probably be properly labelled every time.

Reply:
Thanks, good catch! I updated the PR.